From 6867771b02347a5c0709a293be6c0225e924a9a3 Mon Sep 17 00:00:00 2001
From: Tom Zanussi <tom.zanussi@intel.com>
Date: Tue, 3 Jul 2012 13:07:22 -0500
Subject: [PATCH] perf: add 'libperl not found' warning

If libperl isn't found, display a message to that effect along with
some hints on how to fix it.

Signed-off-by: Tom Zanussi <tom.zanussi@intel.com>
---
 tools/perf/Makefile.config | 1 +
 1 file changed, 1 insertion(+)

diff --git a/tools/perf/Makefile.config b/tools/perf/Makefile.config
index 2916d59c88cd..18cdab0af7c7 100644
--- a/tools/perf/Makefile.config
+++ b/tools/perf/Makefile.config
@@ -837,6 +837,7 @@ else
   FLAGS_PERL_EMBED=$(PERL_EMBED_CCOPTS) $(PERL_EMBED_LDOPTS)
 
   ifneq ($(feature-libperl), 1)
+    msg := $(warning libperl not found, disables Perl scripting support. Please install libperl-dev or perl-devel);
     CFLAGS += -DNO_LIBPERL
     NO_LIBPERL := 1
     $(warning Missing perl devel files. Disabling perl scripting support, please install perl-ExtUtils-Embed/libperl-dev)
-- 
2.39.2

